R/pkg/inst/worker/daemon.R

```diff
@@ -31,7 +31,15 @@ inputCon <- socketConnection(
     port = port, open = "rb", blocking = TRUE, timeout = connectionTimeout)
 
 while (TRUE) {
-  ready <- socketSelect(list(inputCon))
+  ready <- socketSelect(list(inputCon), timeout = 1)
```

Comment:
Yea, I got your point and was taking a look. I think probably feasible via `parallel:::children`. I will give a shot to deal with this here and be back.